src/customizations/volto/components/manage/Contents/Contents.jsx

```diff
@@ -395,6 +432,16 @@ class Contents extends Component {
     this.paste = this.paste.bind(this);
     this.fetchContents = this.fetchContents.bind(this);
     this.orderTimeout = null;
+    this.deleteItemsToShowThreshold = 10;
+    this.filterTimeout = null;
+    this.defaultIndexes = [
+      ...DefaultVoltoIndexes,
+      ...config.settings.customDefaultIndexes,
+    ];
+    this.Indexes = {
+      ...VoltoIndexes,
+      ...config.settings.customIndexes,
+    };
```

Comment:
if(!this.index){this.index={
       order: keys(this.Indexes),
       values: mapValues(this.Indexes, (value, key) => ({
          ...value,
          selected: indexOf(this.defaultIndexes, key) !== -1,
        })),
       selectedCount: this.defaultIndexes.length + 1,
     },
}